src/proof.ml

```diff
@@ -73,30 +73,33 @@ let focus_goal : proof_state -> Env.t * term = fun ps ->
 let pp_goals : _ pp = fun oc gl ->
```

Comment:
This function is also used for printing in the console, so please revert the changes to this file. If you can use arbitrary HTML, why not use the format ``"<pre>%a</pre>"`` when printing with the (unchanged) function?

Reply:
Solved in 9b99f85602779c9d988250295732f7f4f61f1a80.